### PR TITLE
package.json: Update moment to 2.11.2

### LINKED ISSUE #60: Dependencies are out of date, which causes a security issue
For more details see:
https://snyk.io/vuln/npm:moment:20160126

### PR DESCRIPTION
**Pull request by fhemberger**
Fixes #60.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "good-squeeze": "2.x.x",
     "hoek": "2.x.x",
     "json-stringify-safe": "5.0.x",
-    "moment": "2.8.x",
+    "moment": "^2.11.2",
```
Comment (arb) [inline package.json]: We do not use the `^` style anywhere in the hapijs org.

**Comment by fhemberger** [comment]
Had to pin it to 2.11.2 instead of 2.11.x, because afaik 2.11.1 is still vulnerable.

**Comment by AdriVanHoudt** [comment]
2.11.x will install 2.11.2 so what would be the problem?

**Comment by fhemberger** [comment]
Updated to 2.11.x

**Comment by AdriVanHoudt** [comment]
:+1:

**Pull request merged**